src/static/css/2019.css

```diff
@@ -752,6 +752,14 @@ footer {
   padding-right: 20px;
 }
 
+/* padding match with header content for non-mobile viewports */
+@media (min-width: 601px) {
```

Comment:
Not sure that the media query is necessary. Could you overwrite the padding left/right in the original `footer` style?

Reply:
I can but we do 20px for header content on mobile devices, so I just wanted to keep it consistent 60px for large screens and keep 20px for mobile. let me know that's not the expected behavior.

Reply:
Oh ok. The convention we use is to make the desktop styles the default and use media queries for <= 600px to handle mobile styles.

Reply:
got it, thanks!

Since we want to keep header and footer for both internal and index page same, we no longer need this style update.
I'll publish new changes as discussed in the #947 to address few additional issues.